first-android-submission.md

```diff
@@ -0,0 +1,53 @@
+# First Submission of an Android App
+
+#### 🤔 What Happened
+
+If you're submitting your Android app to Google Play Store for the first time you need to do it manually. If you tried to take advantage of the `expo upload:android` you most probably got the following error:
+
+> You haven't submitted this app to Google Play Store yet. The first submission of the app needs to be performed manually.
+
+#### 💡 Solution
+
+This page will guide you through creating an app in Google Play Console and uploading the first release.
+
+1. Open [Google Play Console](https://play.google.com/apps/publish/).
+
+[<img src="./assets/first-android-submission/01-open-google-play-console.png" width="800" />](./assets/first-android-submission/01-open-google-play-console.png)
+
+2. Click the **CREATE APPLICATION** button. A modal will pop up.
+3. Select **Default language**, fill in the **Title** input and click **CREATE**.
+
+[<img src="./assets/first-android-submission/02-create-application.png" width="800" />](./assets/first-android-submission/02-create-application.png)
+
+4. You'll be redirected to a new subpage where you can set **Product details**. **You don't need to set them now.**
+
+[<img src="./assets/first-android-submission/03-product-details.png" width="800" />](./assets/first-android-submission/03-product-details.png)
+
+5. Go to the **App releases** tab and scroll to the **Internal test track** section. Click the **MANAGE** button.
+
+[<img src="./assets/first-android-submission/04-app-releases.png" width="800" />](./assets/first-android-submission/04-app-releases.png)
+
+6. Click the **CREATE RELEASE** button.
+
+[<img src="./assets/first-android-submission/05-create-release.png" width="800" />](./assets/first-android-submission/05-create-release.png)
+
+7. Opt in to app signing by Google Play. Althought it's not necessary, this will let you upload your app when you lose your Android Keystore. Click the **CONTINUE** button.
```

Comment:
```suggestion
7. Opt in to app signing by Google Play. Althought it's not necessary, this will let you upload your app if you somehow happen to lose your Android Keystore. Click the **CONTINUE** button.
```